org.tryton.Tryton-66.desktop

```diff
@@ -0,0 +1,30 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Tryton 6.4
```

Comment:
version up there list 6.6.1. But then the version shouldn't be part of the name in most case.

Reply:
Here it is the branch, we should be 6.6